src/assets/scss/components/_map.scss

```diff
@@ -1,13 +1,13 @@
 
 
 .map-container {
-    height: 30vh;
+    height: 100%;

```

Comment:
Since .map also is height: 100%, the two div's in map.component.html with class "map-container" and "map-frame" could be removed and then the .map-container and .map-frame in this file could be removed.

Reply:
Removed container and frame wrappers